contracts/src/LightClient.sol

```diff
@@ -62,6 +56,15 @@ contract LightClient is Initializable, OwnableUpgradeable, UUPSUpgradeable {
     /// @notice mapping to store light client states in order to simplify upgrades
     mapping(uint32 index => LightClientState value) public states;
```

Comment:
```suggestion
```

we should remove this by now?

Reply:
ah yes thank you, done here https://github.com/EspressoSystems/espresso-sequencer/commit/3f4417012c9b75e967b24de3ae35b02b9c7f504b